### PR TITLE
Solvency fixes

### PR DESCRIPTION
**Pull request by netopyr**
This change adds several small fixes to the solvency checks.

Unfortunately, I had to disable some tests because of an open issue. Once clarified, the tests can be fixed and enabled again.

### CODE REVIEW

**Review by iwsimon — APPROVED** [review]
LGTM

**Review by mhess-swl — APPROVED** [review]

**Review by povolev15 — APPROVED** [review]
LGTM

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/SolvencyPreCheck.java [hunk 1]
```diff
@@ -145,6 +145,9 @@ private long estimateAdditionalCosts(@NonNull final TransactionInfo txInfo, @Non
         return switch (txInfo.functionality()) {
             case CRYPTO_CREATE -> txInfo.txBody().cryptoCreateAccountOrThrow().initialBalance();
             case CRYPTO_TRANSFER -> {
+                if (!txInfo.txBody().cryptoTransferOrThrow().hasTransfers()) {
+                    yield 0L;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/SolvencyPreCheck.java:149]: Why do we need this check ? Wouldn't we catch NPE is there are no transfers and fail with `InsufficientBalanceException(INVALID_TRANSACTION_BODY, totalFees)` ?

Comment (Neeharika-Sompalli) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/SolvencyPreCheck.java:149]: Ah I see. Makes sense. Thanks ! Probably there is a check to see if both `TransferList` and `TokenTransferList` is null. Then it should throw `INVALID_TRANSACTION_BODY`

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM !

**Pull request merged**